src/Java.Interop/Java.Interop/JniRuntime.JniValueManager.cs

```diff
@@ -86,6 +86,8 @@ protected virtual void Dispose (bool disposing)
 
 			public abstract List<JniSurfacedPeerInfo>   GetSurfacedPeers ();
 
+			public abstract void ActivatePeer (IJavaPeerable? self, JniObjectReference reference, Type peerType, Type []? constructorArguments, object? []? argumentValues);
```

Comment:
I take it you didn't like the `ConstructorInfo` idea? https://github.com/xamarin/xamarin-android/pull/5444#issuecomment-761243971

Reply:
I implemented it based on discussion we had last week before reading the comment.

Updated.